week1/homework.md

```diff
@@ -0,0 +1,59 @@
+# Homework week 1
+
+1. Go to `https://github.com/HackYourFuture-CPH/hyf-homework` and fork this repository. Now you should have the `hyf-homework` repository in your own github account.
```

Comment:
I'd add: 
1) that this part of the homework is mandatory 
2) what the purpose of this part  is: "The idea is to fork the homework repository from hack your future, clone it into your computer, do some changes, and push the changes into your fork (copy) of the repository."